README.md

```diff
@@ -292,6 +292,8 @@ Building the program with BLAS support may lead to some performance improvements
     cmake --build . --config Release
     ```
 
+Prompt processing will automatically be GPU accelerated. To enable token generation acceleration use the `-ngl` or `--n-gpu-layers` argument and specify how many layers should be offloaded to the GPU. A higher value will enable more GPU acceleration but also increase VRAM usage. Maximum effective values: 33 for 7b, 41 for 13b, 61 for 33b, 81 for 65b. Multi-GPU setups and iGPUs are currently not supported.
```

Comment:
are those values right? afaik 7B has 32 layers.

Reply:
If the value of GPU layers is higher than the number of repeating layers then the non-repeating layers are also getting offloaded so 33 is still giving you an improvement for 7b compared to 32.